test/iis_shc_issuer/vxu_client_test.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require 'iis_shc_issuer/vxu_client'
+require 'vcr'
+
+VCR.configure do |config|
+  config.cassette_library_dir = 'test/vcr_cassettes/iis_shc_issuer'
+  config.hook_into :webmock
+end
+
+class VXUClientTest < ActiveSupport::TestCase
+  # WARNING: Running tests with this test uncommented could change sandbox data and cause other tests to fail
```

Comment:
This warning is obsolete now, right?

Reply:
Yes, that was left over. Since the tests are mocked, this won't actually affect the Sandbox data anymore. I'll remove.